Sources/PaltaLib/Engine/PaltaLib.swift

```diff
@@ -4,6 +4,13 @@ public final class PaltaLib {
 
     var targets = [Target]()
     var amplitudeInstances = [Amplitude]()
+    public var trackingSessionEvents: Bool = false {
```

Comment:
Could you please add this property to `ConfigurationOptions` struct to prevent using `PaltaLib` without this behavior when we want that behavior to be enabled?

Reply:
updated PR